src/handlers/handler.ts

```diff
@@ -35,12 +36,15 @@ export default class Handler {
     console.log('Disconnected');
   }
 
-  private async disconnectedCb() {
+  private async connectionRetryCb(connectionCb?: ConnectionCallback, disconnectionCb?: DisconnectionCallback, webSocket?: any) {
     try {
+      if (disconnectionCb) {
+        disconnectionCb(webSocket);
```

Comment:
where is the webSocket passed from?

Reply:
it's the last param of connectionRetryCb().

Reply:
https://github.com/ainblockchain/ain-js/blob/master/src/event-manager/event-channel-client.ts#L153

Reply:
it seems that the webSocket object is not being passed in [this line](https://github.com/ainize-team/ainize-js/pull/126/files#diff-c015fc78f932d8b4de4f4a5568abbc291acf823c439ab951dd324089564a29cbR29), but that parameter appears to be required in the ain-js.